sql_prep/models/achievements/agg__prvni_vyhra.sql

```diff
@@ -0,0 +1,37 @@
+WITH base AS (
+    SELECT * FROM {{ ref('base__debater_debata') }}
+),
+
+newbies AS (
+    SELECT * from base
+    RIGHT JOIN {{ ref('base__newbies') }} using (clovek_id)
+),
+
+vyhry AS (
+    SELECT
+        clovek_id,
+        school_year,
+        debate_date,
+    FROM newbies
+    WHERE is_winner = true
+    ORDER BY debate_date
+),
+
+achievement AS (
+    SELECT DISTINCT ON (clovek_id) *
+    FROM vyhry
+),
```

Comment:
This is a fairly punk way of selecting a minimum, but it works so let's not change it :) For future reference, `select clovek_id, min(debate_date) from vyhry group by 1` should do the trick without relying on ordering or `distinct`.

Reply:
I understand and see your code as much better, however I was unable to fix it that way, my attempt is commented, but throws error.